File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        vector<int> sorted=nums;
        sort(sorted.begin(),sorted.end());
        for(int i=0;i<nums.size();i++){
            int found=1;
            for(int j=0;j<nums.size();j++){
                if(sorted[j]!=nums[(j+i)%nums.size()]){
                    found=0;
                    break;
                }
            }
            if(found==1)return true;
        }
        return false;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int rotated=0;
        for(int i=0;i<nums.length-1;i++){
            if(nums[i]>nums[i+1])rotated++;
        }
        return (rotated==1 && nums[nums.length-1]<=nums[0]) || rotated==0;
    }
}